### PR TITLE
Added ingest Vulnerability for both backends and cmd test data

### PR DESCRIPTION
**Pull request by pxp928**
- fix wrong variable name for `IsOccurrenceSpecInputSpec` to `IsOccurrenceInputSpec`
- fix wrong edge name on `certifyVuln`
- added ingest Vulnerability for testing and neo4j backends
- added ingestion via cmd

### CODE REVIEW

File: cmd/graphql_playground/cmd/ingest.go
Comment (pxp928) [inline cmd/graphql_playground/cmd/ingest.go:207]: getting the following error while ingesting into neo4j:
```
{"level":"error","ts":1677966566.181216,"caller":"cmd/ingest.go:263","msg":"Error in ingesting: input: ingestVulnerability Neo4jError: Neo.ClientError.Statement.TypeError (Unable to construct ZonedDateTime value: `Unknown time-zone ID: Local`)\n\n","stacktrace":"github.com/guacsec/guac/cmd/graphql_playground/cmd.IngestVulnerability\n\t/Users/parth/Documents/pxp928/artifact-ff/cmd/graphql_playground/cmd/ingest.go:263\ngithub.com/guacsec/guac/cmd/graphql_playground/cmd.ingestData\n\t/Users/parth/Documents/pxp928/artifact-ff/cmd/graphql_playground/cmd/ingest.go:45"}
```
What was the solution to this?
Keeping this comment to block merge!

Comment (lumjjb) [reply cmd/graphql_playground/cmd/ingest.go:207]: this wasn't the same error as the one i encountered - that was more for the missing values, this seems like maybe a formatting issue, maybe on the neo4j ingestion side setting the output of the value encoded to RFC3339 format maybe?

Comment (mihaimaruseac) [reply cmd/graphql_playground/cmd/ingest.go:207]: I have encountered the same error, it's because the backend expects a different format for the time field.

I'm working on that

**Review by mihaimaruseac — APPROVED** [review]

File: cmd/graphql_playground/cmd/ingest.go
Comment (pxp928) [reply cmd/graphql_playground/cmd/ingest.go:207]: Keeping this open to make sure all the ingestion and query occur properly before merging.

**Review by lumjjb — APPROVED** [review]

File: pkg/assembler/backends/neo4j/certifyVuln.go
Comment (mihaimaruseac) [inline pkg/assembler/backends/neo4j/certifyVuln.go:399]:
```suggestion
	queryValues[timeScanned] = certifyVuln.TimeScanned.UTC()
```

Based on #538. We need to make sure times are in UTC to not confuse neo4j.

File: pkg/assembler/graphql/schema/certifyVuln.graphql
Comment (mihaimaruseac) [inline pkg/assembler/graphql/schema/certifyVuln.graphql:46]: Let's remove this, I'm adding it in #538 with comments

File: cmd/graphql_playground/cmd/ingest.go
Comment (mihaimaruseac) [reply cmd/graphql_playground/cmd/ingest.go:207]: Fixing in #538 , the issue was actually in a mismatch between Go's default timezone in `time.Now()` and neo4j's custom type for local timezones. Switching to UTC solves this and also the fact that our times need to be localized with some timezone.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: cmd/graphql_playground/cmd/ingest.go
Comment (pxp928) [reply cmd/graphql_playground/cmd/ingest.go]: fixed.

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**
